efel/cppcore/LibV5.cpp

```diff
@@ -3626,6 +3633,121 @@ int LibV5::postburst_min_values(mapStr2intVec& IntFeatureData,
   return 1;
 }
 
+static int __postburst_slow_ahp_indices(vector<double>& t,
+                                   vector<double>& v,
+                                   vector<int>& peak_indices,
+                                   vector<int>& burst_end_indices,
+                                   vector<int>& postburst_slow_ahp_indices,
+                                   vector<double>& postburst_slow_ahp_values,
+                                   const double stim_end,
+                                   double sahp_start) {
+  unsigned postburst_slow_ahp_index, stim_end_index, end_index, t_start_index;
+  stim_end_index =
+      distance(t.begin(),
+                find_if(t.begin(), t.end(),
+                        std::bind2nd(std::greater_equal<double>(), stim_end)));
+  end_index = distance(t.begin(), t.end());
+  for (size_t i = 0; i < burst_end_indices.size(); i++) {
+    double t_start = t[peak_indices[burst_end_indices[i]]] + sahp_start;
+
+    if (burst_end_indices[i] + 1 < peak_indices.size()){
+      t_start_index = find_if(
+        t.begin() + peak_indices[burst_end_indices[i]],
+        t.begin() + peak_indices[burst_end_indices[i] + 1],
+        std::bind2nd(std::greater_equal<double>(), t_start)
+      ) - t.begin();
+      postburst_slow_ahp_index = min_element(
+        v.begin() + t_start_index,
+        v.begin() + peak_indices[burst_end_indices[i] + 1]
+      ) - v.begin();
+    } else if (peak_indices[burst_end_indices[i]] < stim_end_index){
+      t_start_index = find_if(
+        t.begin() + peak_indices[burst_end_indices[i]],
+        t.begin() + stim_end_index,
+        std::bind2nd(std::greater_equal<double>(), t_start)
```

Comment:
I think `std::bind2nd` is deprecated in C++11 and removed in C++17.
https://en.cppreference.com/w/cpp/utility/functional/bind12

There are lambda expressions available now for the replacement.
https://en.cppreference.com/w/cpp/language/lambda

```
t_start_index = find_if(
    t.begin() + peak_indices[burst_end_indices[i]],
    t.begin() + end_index,
    [t_start](double value) { return value >= t_start; }
) - t.begin();
```

Reply:
thanks, solved in last commit